python_modules/libraries/dagster-shared/dagster_shared/plus/config.py

```diff
@@ -67,6 +67,64 @@ def get(cls) -> "DagsterPlusCliConfig":
         _, _, raw_plus_config, _ = result
         return cls(**raw_plus_config)
 
+    @classmethod
+    def create_for_deployment(
+        cls,
+        deployment: Optional[str],
+        organization: Optional[str] = None,
+        user_token: Optional[str] = None,
+    ) -> "DagsterPlusCliConfig":
+        """Create a DagsterPlusCliConfig instance for deployment-scoped operations.
+
```

Comment:
was this and the file below supposed to be included here? i don't see callsites

Reply:
its used upstack